operate/client/e2e-playwright/pages/components/Diagram.ts

```diff
@@ -49,4 +49,17 @@ export class Diagram {
       .locator('.djs-element')
       .filter({hasText: new RegExp(`^${flowNodeName}$`, 'i')});
   }
+
+  getExecutionCount(elementId: string) {
+    return this.diagram.evaluate(
+      (node, {elementId}) => {
+        const completedOverlay: HTMLDivElement | null = node.querySelector(
+          `[data-container-id="${elementId}"] [data-testid="state-overlay-completed"]`,
+        );
```

Comment:
I couldn't find a better way to get the number from the badges. Open to ideas, if there is a better way.

Reply:
No idea how to do this in a cleaner way as well, for now it should be fine 😅